$NetBSD: patch-src_Makefile,v 1.2 2015/05/20 23:08:37 khorben Exp $

Do not try to build targets that are created by existing targets, causes
problems with parallel builds and is unnecessary.

--- src/Makefile.orig	2015-05-10 22:19:30.000000000 +0000
+++ src/Makefile
@@ -1,4 +1,4 @@
-TARGETS	= $(OBJDIR)libSystem.a $(OBJDIR)libSystem.so.0.0 $(OBJDIR)libSystem.so.0 $(OBJDIR)libSystem.so
+TARGETS	= $(OBJDIR)libSystem.a $(OBJDIR)libSystem.so.0.0
 OBJDIR	=
 PREFIX	= /usr/local
 DESTDIR	=
